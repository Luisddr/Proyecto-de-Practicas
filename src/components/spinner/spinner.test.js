import ShallowRenderer from "react-test-renderer/shallow"
import Spinner from "./spinner";
import React from 'react';


const renderer = new ShallowRenderer();

it('expect to render Spinner', ()=>{
    
    // const result = renderer.getRenderOutput()
    expect(renderer.render(<Spinner/>)).toMatchSnapshot()
})