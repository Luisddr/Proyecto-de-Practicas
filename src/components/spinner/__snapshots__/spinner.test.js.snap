// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Spinner 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;
